packages/block-library/src/query/style.scss

```diff
@@ -24,6 +24,16 @@
 			300ms
 			ease-in;
 	}
+
+	&.start-animation,
+	&.finish-animation {
+		@media (prefers-reduced-motion: reduce) {
```

Comment:
Can you double check that we have to remove the loading bar animation on reduced motion? I thought that was only necessary for the fading animation:

- https://github.com/WordPress/gutenberg/issues/54060

Reply:
Good call, I don't know if that's necessary or not.

Reply:
> I thought that was only necessary for the fading animation:

Yeah. I remember the reduced motion thing was mentioned, but as it was not included in the list of enhancements, I think I applied it to the wrong animation... 😅 

Whatever you decide is good, @jasmussen. I can revert that change.

Reply:
Reverted in https://github.com/WordPress/gutenberg/pull/54455/commits/0f163c4dfbccfd83cee9993d7ecfee90863861ef.